### ISSUE TITLE
Accessibility issues in fd-table

### ISSUE THREAD

**Issue by m8524769** [SAP/fundamental-ngx#11211]
#### Is this a bug, enhancement, or feature request?

bug

#### Describe your proposal.

Case 1:

On accessing the selectable entire row, the corresponding table column headers should be announced along with the data cells.

Case 2:

On accessing the check box within a table column, its UI type should be announced as check box.

Case 3:

If we embed a fd-table in a fd-dialog's body, then the content of the table should not be announced automatically when the dialog is opened.

#### Can you handle that on the application side

no

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

0.43.11

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

Create a fd-table with checkbox and enable focusable for both rows and cells, then TAB into the table.

#### Did you check the documentation and the API?

yes

#### Did you search for similar issues?

yes

**Comment by droshev** [comment]
@m8524769 can you try with `0.43.25`

**Comment by droshev** [comment]
no activities => closing the issue
